code/procs/gamehelpers.dm

```diff
@@ -776,3 +776,81 @@ proc/get_ouija_word_list(var/atom/movable/source = null, var/words_min = 5, var/
 					words |= (M.real_name ? M.real_name : M.name)
 
 	return words
+
+
+// returns initial health of an item or an item type
+/proc/get_initial_item_health(obj/item/I)
+	if (initial(I.health))
+		return initial(I.health)
+	else
+		var/weight_class = initial(I.w_class)
+		switch (weight_class)
+			if (W_CLASS_TINY to W_CLASS_NORMAL)
+				return weight_class + 1
+			else
+				return weight_class + 2
```

Comment:
?

Reply:
Oh I see what you've done, probably best to revert to your first commit and then merge again and select "accept both"

Reply:
reverted, not sure if I did what you said to do correctly though?

Reply:
yep, now you need to merge again and keep both upstream and local changes